03 Asset Classes/06 Crypto/01 Crypto/content.html

```diff
@@ -49,11 +49,11 @@ <h3>Live Trading</h3>
 		<br/>
 		Return to QuantConnect.com and go to the algorithm that you want to trade live. Select, 'Go Live' from the upper right hand corner. Follow the 'Live' wizard and select 'Gdax' as your brokerage. Be sure to copy your api key, api secret and api passcode correctly in the wizard.
 		<br/>
-		Once potential stumbling block that users face deploying their algorithms live with GDAX is misunderstanding what type of account they have with GDAX. Typically, when users first open their account with GDAX, they are given a Cash account. In other words, margin trading is not available. However, margin accounts are the default account type specified in QCAlgorithm. Therefore, when users try to place trades using methods like <pre>SetHoldings()</pre> they will get errors that crash their algorithm. Be sure to set your account type using the <pre>SetBrokerage()</pre> method.
+		Once potential stumbling block that users face deploying their algorithms live with GDAX is misunderstanding what type of account they have with GDAX. Typically, when users first open their account with GDAX, they are given a Cash account. In other words, margin trading is not available. However, margin accounts are the default account type specified in QCAlgorithm. Therefore, when users try to place trades using methods like <div class="section-example-container"><pre>SetHoldings()</pre></div> they will get errors that crash their algorithm. Be sure to set your account type using the <div class="section-example-container"><pre>SetBrokerage()</pre></div> method.
```

Comment:
"<div class="section-example-container"><pre>SetHoldings()</pre></div>"

Why would this be a pre?? This should be <code> @LiXiang618

Reply:
And should also remove the <div class="section-example-container">